wicket-jquery-ui/src/main/java/com/googlecode/wicket/jquery/ui/interaction/sortable/Sortable.java

```diff
@@ -168,7 +175,8 @@ public boolean isOnReceiveEnabled()
 	@Override
 	public boolean isOnRemoveEnabled()
 	{
-		return false; // 'remove' will be handled after 'receive' by the event bus because there is a risk the item is removed before being received (leading to a NPE)
+		return false; // 'remove' will be handled after 'receive' by the event bus because there is a risk the item is removed before being received
+						// (leading to a NPE)
```

Comment:
comment broken, I will rewrite it anyway because it is not as clear in my head as the time I wrote it :)